tests/openvino/test_exporters_cli.py

```diff
@@ -325,6 +385,58 @@ def test_exporters_cli_tokenizers(self, task: str, model_type: str):
 
+    # some testing models required transformers at least 4.45 for conversion
+    @parameterized.expand(TOKENIZER_CHAT_TEMPLATE_TESTS_MODELS)
+    @unittest.skipIf(
+        is_transformers_version("<", "4.45.0") or not is_openvino_tokenizers_available(),
+        reason="test required openvino tokenizers and transformers >= 4.45",
+    )
+    def test_exporters_cli_tokenizers_chat_template(self, model_type):
+        import openvino as ov
+
+        core = ov.Core()
+        with TemporaryDirectory() as tmpdir:
+            model_test_config = self.TOKENIZER_CHAT_TEMPLATE_TESTS_MODELS[model_type]
+            task = model_test_config["task"]
+            model_id = MODEL_NAMES[model_type]
+            remote_code = model_test_config.get("remote_code", False)
+            cmd = f"TRANSFORMERS_VERBOSITY=debug optimum-cli export openvino --model {model_id} --task {task} {tmpdir}"
+            if remote_code:
+                cmd += " --trust-remote-code"
+            output = subprocess.check_output(
+                cmd,
+                shell=True,
+                stderr=subprocess.STDOUT,
+            ).decode()
+            number_of_tokenizers = sum("tokenizer" in file for file in map(str, Path(tmpdir).rglob("*.xml")))
+            expected_num_tokenizers = model_test_config["num_tokenizers"]
+            self.assertEqual(expected_num_tokenizers, number_of_tokenizers, output)
+            tokenizer_path = (
+                Path(tmpdir) / "openvino_tokenizer.xml"
+                if "diffusion" not in model_type
+                else Path(tmpdir) / "tokenizer/openvino_tokenizer.xml"
+            )
+            tokenizer_model = core.read_model(tokenizer_path)
+            if not model_test_config.get("expected_chat_template", False):
+                self.assertFalse(tokenizer_model.has_rt_info("chat_template"))
+            else:
+                rt_info_chat_template = tokenizer_model.get_rt_info("chat_template")
+                if not model_test_config.get("processor_chat_template"):
+                    ref_chat_template = AutoTokenizer.from_pretrained(
+                        tmpdir, trust_remote_code=remote_code
+                    ).chat_template
+                else:
+                    ref_chat_template = AutoProcessor.from_pretrained(
+                        tmpdir, trust_remote_code=remote_code
+                    ).chat_template
+                self.assertEqual(rt_info_chat_template, ref_chat_template)
+                if not model_test_config.get("simplified_chat_template", False):
+                    self.assertFalse(tokenizer_model.has_rt_info("simplified_chat_template"))
+                else:
+                    simplified_rt_chat_template = tokenizer_model.get_rt_info("simplified_chat_template")
+                    self.assertTrue(rt_info_chat_template in COMPLEX_CHAT_TEMPLATES)
+                    self.assertEqual(simplified_rt_chat_template, COMPLEX_CHAT_TEMPLATES[rt_info_chat_template])
```

Comment:
maybe we can run some tests that `simplified_rt_chat_template` and original one lead to the same output?
E.g. run python Jinja on some test inputs and compare outputs

Reply:
added